.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Issue type
```

Comment:
@noor-ahmed How about adding 2 templates for the bug report and feature request? as same as what we have on [Rails template](https://github.com/nimblehq/rails-templates/tree/develop/.github/ISSUE_TEMPLATE) and [Gin template](https://github.com/nimblehq/gin-templates/tree/develop/.github/ISSUE_TEMPLATE) 🤔

Reply:
Added two template here 🙌🏻  2a60878